cross/squid/PLIST

```diff
@@ -1,75 +1,36 @@
-bin:sbin/squid
-etc:etc/cachemgr.conf
-etc:etc/cachemgr.conf.default
-etc:etc/errorpage.css
-etc:etc/errorpage.css.default
-etc:etc/mime.conf
-etc:etc/mime.conf.default
-etc:etc/msntauth.conf
-etc:etc/msntauth.conf.default
-etc:etc/squid.conf
-etc:etc/squid.conf.default
-etc:etc/squid.conf.documented
+bin:bin/purge
+bin:bin/squidclient
+rsc:libexec/basic_db_auth
+bin:libexec/basic_fake_auth
```

Comment:
Choose the appropriate prefix based on the dir name: Use `libexec` here, not `bin`.